src/Transmailifier/Mailer.php

```diff
@@ -74,7 +66,7 @@ public function notify(array $transactions, string $description, array $addresse
         try {
             $this->mailer->send($message);
         } finally {
-            unlink($file);
+            // unlink($file);
```

Comment:
Make the mail sending process instant (no delay) so the file can be cleaned up here.